src/NServiceBus.Core/Audit/Audit.cs

```diff
@@ -10,7 +10,11 @@ public class Audit : Feature
         internal Audit()
         {
             EnableByDefault();
-            Prerequisite(config => AuditConfigReader.GetConfiguredAuditQueue(config.Settings, out auditConfig), "No configured audit queue was found");
+            Defaults(settings =>
+            {
+                settings.Set<AuditConfigReader.Result>(AuditConfigReader.GetConfiguredAuditQueue(settings));
```

Comment:
Since we are now reading the audit config potentially earlier could we run into issues with IProvideConfiguration and order of execution?